server.js

```diff
@@ -7,11 +7,11 @@
 /////////////////////////////
 
 var port = process.env.PORT || 3000;
-
+var db = require('./models');
 //require express in our app
 var express = require('express'),
   bodyParser = require('body-parser');
```

Comment:
You need to require mongoose and configure dotenv here in order to use them below. 

```js
const mongoose = require('mongoose');
require('dotenv').config();
```